src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const clone = {};
+
+  for (const key in robot) {
+    const newKey = robot[key];
+
+    if (clone.hasOwnProperty(newKey)) {
```

Comment:
The method 'hasOwnProperty' is a safe way to check for property existence, but it's recommended to use 'Object.prototype.hasOwnProperty.call(clone, newKey)' to avoid issues in cases where the object might have a property with the name 'hasOwnProperty'.